README.md

```diff
@@ -1 +1,16 @@
 GOOGLE CLONE PROJECT (LUIZ, HAKEEM, ANWAR)
+
+We could develop the API's using the routes. 
+We couldnt find a way of linking the routes to the google button...
+Should we have used the fetch aproach? Is fetch the best way of doing it when we need to use the data response to manipulate de DOM?
+
+
+how to use the data response from the endpoints ?
+Should I store it in a variable? How?
+I guess it can be done by using fetch but still dont see how to implement it...
+Is there any other way of implementing this using the routes on our index.js?
+
+
+After a lot of google searches I couldnt see becouse most of the examples Ive found was based in fetching one url rather than an array of objects.
+
```

Comment:
Could have used a Client and a Server folder to organise the files more whilst having another JS file which interacts with the html file.